src/components/Player.jsx

```diff
@@ -19,14 +19,14 @@ export default function Player(props) {
         <div className="cursor-pointer relative">
             {props.team === "wufo" ? 
               <img 
-                src={require(`../assets/wufopics/roster/${props.name}.jpg`).default}
-                alt={`${props.name}`}
+                src={require(`../assets/wufopics/roster/${props.last.toLowerCase()}.jpg`).default}
```

Comment:
Might become a problem in the future if there's ever multiple players with the same last name, but I guess whoever is dealing with that can just make the image names be `props.first-props.last.jpg` or use the jersey number instead.

Reply:
I'll make that change